README.md

```diff
@@ -4,7 +4,8 @@
 
 ---
 
-[![Documentation (master)](https://img.shields.io/badge/docs-master-59f)](https://libra.github.io/libra/)
+[![Developer Documentation (master)](https://img.shields.io/badge/docs-master-59f)](https://developers.libra.org/)
+[![Libra Rust Crate Documentation (master)](https://img.shields.io/badge/docs-master-59f)](https://developers.libra.org/docs/rustdocs/)
```

Comment:
We either need to remove the Developer Doc badge or change the url because right now these two are rendered identically

Reply:
Should I create a new badge for the site itself, you think?

Reply:
I don't know if we necessarily need a badge for the website itself but if you think it would be valuable then go ahead :)

Reply:
I will add it later, once we land this. 👍